shared/src/main/scala/mlscript/JSBackend.scala

```diff
@@ -356,16 +370,19 @@ class JSBackend(pgrm: Pgrm) {
 
   def translateClassMember(
       method: MethodDef[Left[Term, Type]]
-  ): JSClassMemberDecl = {
+  )(implicit scope: Scope): JSClassMemberDecl = {
     val name = method.nme.name
     method.rhs.value match {
-      case Lam(Var(param), rhs)         => JSClassMethod(name, JSNamePattern(param) :: Nil, L(translateTerm(rhs)))
-      case Lam(Asc(Var(param), _), rhs) => JSClassMethod(name, JSNamePattern(param) :: Nil, L(translateTerm(rhs)))
-      case Lam(Tup(params), body) => JSClassMethod(
-        name,
-        params map { case _ -> param => translatePattern(param) },
-        L(translateTerm(body))
-      )
+      case Lam(Var(param), rhs) =>
+        JSClassMethod(name, JSNamePattern(param) :: Nil, L(translateTerm(rhs)))
+      case Lam(Asc(Var(param), _), rhs) =>
```

Comment:
I don't understand why this case and the one above needs special handling. Why not handle everything with `translatePattern`? There is nothing special about ascribing _one_ parameter (as opposed to any other number of parameters).

Reply:
`translatePattern` treats single parameter. But I do found similar patterns in  `translateTerm` and `translateClassMember`. I extract a new function for translating parameter lists.